OpenRA.Mods.Common/Traits/Player/PlaceBuilding.cs

```diff
@@ -143,11 +146,28 @@ public void ResolveOrder(Actor self, Order order)
 				}
 
 				if (GetNumBuildables(self.Owner) > prevItems)
-					w.Add(new DelayedAction(info.NewOptionsNotificationDelay,
-							() => Game.Sound.PlayNotification(self.World.Map.Rules, order.Player, "Speech", info.NewOptionsNotification, order.Player.Faction.InternalName)));
+				{
+					this.order = order;
```

Comment:
We already know the `order.Player` (it's `self.Owner`), so we shouldn't need to store this.